.github/workflows/docker-backend.yml

```diff
@@ -34,17 +29,10 @@ jobs:
   docker:
     runs-on: ubuntu-latest-16core
     env:
-      # conditionally adds '/' if inputs.name isn't empty (https://github.com/orgs/community/discussions/25725)
-      IMAGE_NAME: "${{ inputs.registry }}/${{ inputs.repository }}${{ inputs.name && '/' || '' }}${{ inputs.name }}"
+      IMAGE_NAME: "${{ inputs.registry }}/backend/${{ inputs.name }}"
```

Comment:
I don't think `IMAGE_NAME` is a good name for this variable anymore, as it ultimately represents the full path. Maybe `IMAGE_PATH`?